sdk/go.mod

```diff
@@ -4,5 +4,6 @@ go 1.13
 
 require (
 	github.com/pkg/errors v0.9.1
+	github.com/pulumi/pulumi/sdk v1.14.1
```

Comment:
looks like this was pulled in -`pulumiManifest.go` ?

Reply:
Yeah, fixed that 👍